common/djangoapps/student/admin.py

```diff
@@ -147,21 +151,16 @@ class Meta(object):
         model = CourseEnrollment
 
 
-class UserProfileAdmin(admin.ModelAdmin):
-    """ Admin interface for UserProfile model. """
-    list_display = ('user', 'name',)
-    raw_id_fields = ('user',)
```

Comment:
Nope. There is one inline displayed on the User detail/edit pages. No more dropdowns or UserProfile views.